crates/types/src/net/mod.rs

```diff
@@ -85,26 +121,24 @@ pub enum BindAddress {
     /// Unix domain socket
     #[display("unix:{}", _0.display())]
     Uds(PathBuf),
-    /// Socket addr.
+    /// Socket address (IP and port).
```

Comment:
I believe this is incorrect, this would be a Unix filesystem path, e.g. `unix:/var/restate/node-ctrl.sock`. Maybe?

```suggestion
    /// Unix socket path.
```

Reply:
I thought of it as 
`unix:/var/restate/node-ctrl.sock` for UDS  `Uds(PathBuf)` 
 `0.0.0.0:5122` for socket `Socket(SocketAddr)`

Reply:
No worries :D